.github/workflows/carp.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-haskell@v1
+
+    - uses: actions/cache@v1
+      name: Cache ~/.stack
+      with:
+        path: ~/.stack
+        key: ${{ runner.os }}-stack
```

Comment:
Caches are immutable meaning that future changes to.`~/.stack` will not be stored.

Reply:
Ah, thanks. I'll look into that.

Reply:
@bjorn3 This should fix it:
https://github.com/carp-lang/Carp/commit/f081992830618565c2523767d9ad0abd9fa07141